Packs/Netmiko/Integrations/Netmiko/Netmiko.yml

```diff
@@ -0,0 +1,306 @@
+category: Utilities
+commonfields:
+  id: Netmiko
+  version: -1
+configuration:
+- display: Platform
+  name: platform
+  required: true
+  type: 0
+- display: Hostname
+  name: hostname
+  required: true
+  type: 0
+- display: Port
+  name: port
+  required: true
+  type: 0
+- display: Credentials
+  name: credentials
+  required: true
+  type: 9
```

Comment:
```suggestion
  type: 4
```
Please use encrypted type in sensitive parameters

Reply:
> Please use encrypted type in sensitive parameters

@ChanochShayner The credentials ARE an encrypted type. They are the built-in type that the platform provides for credentials:

![image](https://user-images.githubusercontent.com/53576129/104911347-a1329780-5982-11eb-83f5-1abe97ff7b2e.png)

Please redact the change.

Reply:
@aburt-demisto You're right, my mistake.

Reply:
@ChanochShayner No worries. I've made all other changes.